docs/en/latest/debug-mode.md

```diff
@@ -36,6 +36,7 @@ You can enable the basic debug mode by adding this line to your debug configurat
 ```yaml title="conf/debug.yaml"
 basic:
   enable: true
+#END
```

Comment:
why add this `#END` in the md file?

Reply:
It was Nicholas doc PR update which went into the 3.7 branch and should be included in the master branch as well.

It might have been accidentally included in this PR, but since the change does need to go into master, I'm ok with it as long as we update the title and description accordingly to reflect this PR ports two doc updates that were already merged in 3.7.